### PR TITLE
Buildit

### PR DESCRIPTION
**Pull request by arjorb**
I am requesting for code review

### CODE REVIEW

File: components/home/Properties.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <div>
+            <h3 className='text-md text-steelBlue-100'>Available Properties</h3>
+            <h1 className='text-4xl font-semibold text-steelBlue-200 mt-7'>Featured Listing</h1>
+            <p className='text-sm text-steelBlue-200 leading-loose  my-7'>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc elit sem, vestibulum sed leo eu, malesuada
+              pharetra mauris. Integer rhoncus, eros vel euismod tempor, ipsum magna tristique nisi, quis ullamcorper
+              enim magna eu orci. Sed semper ex quis semper aliquet. Cras hendrerit molestie sapien sed fermentum.
+              Mauris dui tortor, viverra vel ultrices in, congue sed ex.
+            </p>
+            <Button name='Explore' />
```
Comment (Stevenkwizera06) [inline components/home/Properties.js]: work on this button

**Review by Stevenkwizera06 — APPROVED** [review]
Nice work john

File: components/about/Card.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Image from 'next/image';
+
+const Card = ({ image }) => {
+  return (
+    <>
+      <div className='w-[250px] h-auto bg-steelBlue-50 shadow-2xl shadow-blue-300 rounded-md'>
```
Comment (MugaboDenys) [inline components/about/Card.js]: You're Still remaining with albitrary values like here

**Review by MugaboDenys — CHANGES_REQUESTED** [review]

**Review by Morrism1 — APPROVED** [review]
LGTM!

**Pull request merged**